### PR TITLE
remove gradle and add liquid

### PR DESCRIPTION
**Pull request by SaketGupte**

### CODE REVIEW

File: Build.Dockerfile [hunk 1]
```diff
@@ -27,6 +26,14 @@ RUN apt-get update \
     \
     && rm -r /var/lib/apt/lists/*
 
+# Install Ruby and RVM
+ RUN gpg --keyserver hkp://keys.gnupg.net:80 --keyserver-options http-proxy=${http_proxy} --recv-keys=${RECV_KEYS} && \
+     curl -sSL https://get.rvm.io | grep -v __rvm_print_headline | bash -s stable --ruby=ruby-2.7.2 && \
+     echo "source /usr/local/rvm/scripts/rvm" >> ~/.bashrc && \
+     # Install gems \
+     /bin/bash -l -c "gem install bundler liquid-cli:0.0.1 stf-client:0.3.0-rc.12 --no-document" && \
+     /bin/bash -l -c "gem update --system"
```
Comment (Raikerian) [inline Build.Dockerfile]: Why not to make a separate image for this? Most of the builds that use Build image won't need this, and those that need this line don't use android sdk?

File: CurlAndLiquid.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            curl \
+            ca-certificates \
+            unzip \
+            gnupg \
+            dirmngr \
+    \
+    && rm -r /var/lib/apt/lists/*
+
+# Install Ruby and RVM
+RUN gpg --keyserver hkp://keys.gnupg.net:80 --keyserver-options http-proxy=${http_proxy} --recv-keys=${RECV_KEYS} && \
```
Comment (Raikerian) [inline CurlAndLiquid.Dockerfile]: since we already use the style in this repo, please keep ampersands on the new line, instead of at the end

**Review by Raikerian — DISMISSED** [review]
Just add new line in liquid docker file

**Comment by SaketGupte** [comment]
Done

**Comment by Raikerian** [comment]
Don't forget to delete the branch after merge

**Pull request merged**